src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -87,11 +87,12 @@ seastar::future<> ClientRequest::start()
 	return interruptor::with_interruption([this, pgref]() mutable {
           epoch_t same_interval_since = pgref->get_interval_start_epoch();
           logger().debug("{} same_interval_since: {}", *this, same_interval_since);
-          return with_sequencer(
-            interruptor::wrap_function([this, pgref]() -> interruptible_future<> {
+          auto interruptible_do_op =
```

Comment:
nit: changing `return with_sequencer` to `auto interruptible_do_op = ...` belongs to the previous commit, I think.

Reply:
fixed.